bundles/org.openhab.binding.magentatv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,258 @@
+			<channel id="duration" typeId="programDuration" />
+			<channel id="position" typeId="programPosition" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Play Status</label>
+		<description>Status information on media play</description>
+		<channels>
+			<channel id="channelCode" typeId="channelCode" />
+			<channel id="runStatus" typeId="runStatus" />
+			<channel id="playMode" typeId="playMode" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="stop">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops the player. ON if the player is stopped.</description>
+	</channel-type>
+	<channel-type id="channel">
+		<item-type>String</item-type>
+		<label>Channel</label>
+		<description>Send channel number to switch program</description>
+	</channel-type>
+	<channel-type id="channelCode" advanced="true">
+		<item-type>String</item-type>
+		<label>Channel Code</label>
+		<description>Channel code in the channel list</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="runStatus" advanced="true">
+		<item-type>String</item-type>
+		<label>Status</label>
+		<description>Run status</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="playMode">
+		<item-type>String</item-type>
+		<label>Play mode</label>
+		<description>Play mode for running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programTitle">
+		<item-type>String</item-type>
+		<label>Program</label>
+		<description>Running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programText">
+		<item-type>String</item-type>
+		<label>Description</label>
+		<description>Some info on the running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programStart">
+		<item-type>String</item-type>
+		<label>Start</label>
+		<description>Program start time</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programDuration">
+		<item-type>String</item-type>
+		<label>Duration</label>
+		<description>Duration of the program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programPosition">
+		<item-type>String</item-type>
+		<label>Play Position</label>
+		<description>Play position since program started</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="channelUp">
+		<item-type>Switch</item-type>
+		<label>Channel up</label>
+		<description>Next channel</description>
+	</channel-type>
+	<channel-type id="channelDown">
+		<item-type>Switch</item-type>
+		<label>Channel down</label>
+		<description>Previous channel</description>
+	</channel-type>
```

Comment:
I'd argue that it is more appropriate to use a write-only command channel to send this information. Since you can already do this in your `key` channel, including `channelUp` and `channelDown` switches seems redundant.

Reply:
removed channels